### ISSUE TITLE
Rating - Ensure parts of graphical objects essential for understanding content have sufficient contrast - (2036570023)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7799]
### Violation: 
 Ensure parts of graphical objects essential for understanding content have sufficient contrast 

![image](https://github.com/Esri/calcite-design-system/assets/5023024/f938e6d4-9592-484c-96f3-2121358785ac)

### WCAG Reference: 
- [1.4.11 Non-test Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) 

### Severity: 
 5 

### Media Type: 
 Color and Contrast 

--- 
### Areas for Remediation: 
- Title: Rating - Ensure parts of graphical objects essential for understanding content have sufficient contrast - (2036570023) 
- Module: Rating 
**Issue**
There are graphical objects with a color contrast ratio below 3.00:1.

Foreground: #A0A0A0
Background: #A0A0A0
Contrast ratio: 2.46:1

Examples include:
- Border of rating star image

**User Impact**
Users with low vision will have difficulty identifying this content.

**Code Reference** 
 ```
<svg class="svg" fill="currentColor" height="100%" viewBox="0 0 16 16" width="100%" xmlns="http://www.w3.org/2000/svg">
...
</svg>

[CSS]
.star {
      outline-color: transparent;
      color: var(--calcite-ui-border-input);
}
 ``` 
**Suggestion**
Ensure parts of graphical objects essential for understanding content have sufficient contrast. The required minimum contrast ratio is 3.00:1.

Common examples of qualifying objects include lines in a chart, meaningful icons, and annotations within an image.

Graphics that require particular presentation to preserve their meaning are exempt from this requirement.

To evaluate color contrast, see the Accessible Color Picker extension: https://www.accessibility.dev/

 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Duplicate of https://github.com/Esri/calcite-design-system/issues/7822.
